### DISCUSSION TITLE
make it so in survival people can read lecterns but not take the books out of them in claims

### DISCUSSION THREAD

**Discussion by NathanMichaelPersonal** [GriefPrevention/GriefPrevention#1739] [Ideas]
make it so in survival people can read lecterns but not take the books out of them in claims, it'll be more fun and we can all make libraries

**Comment by Jikoo** [comment]
Lectern read access is possible with AccessTrust. Book removal requires ContainerTrust. Simply `/AccessTrust public` in the claim you want to make your library area.
